### PR TITLE
An extension to quickly create latex for your documents

### PR DESCRIPTION
**Pull request by Noamko**
## Description

This extension enables users to input any LaTeX text, which will be parsed into a LaTeX image and copied to the clipboard.

## Screencast

 [New Recording - 6/14/2023, 12:52:54 AM](https://vimeo.com/835964536/cba96d909d)

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder
 [New Recording - 6/14/2023, 12:52:54 AM](https://vimeo.com/835964536/cba96d909d)

### CODE REVIEW

File: extensions/quick-latex/src/quick-latex.tsx [hunk 1]
```diff
@@ -0,0 +1,56 @@
+import { Action, ActionPanel, environment, List, popToRoot, showHUD } from "@raycast/api";
+import { useState, useEffect } from "react";
+import { existsSync, mkdirSync } from "fs";
+import { resolve } from "path";
+import { runAppleScript } from "run-applescript";
+import { image } from "image-downloader";
+
+const downloadDir = resolve(environment.supportPath, "download");
+const latexUrl = "https://latex.codecogs.com/png.image?" + encodeURIComponent("\\dpi{512}");
+const latexUrlDark = "https://latex.codecogs.com/png.image?" + encodeURIComponent("\\dpi{512}\\bg{white}");
```
Comment (pernielsentikaer) [inline extensions/quick-latex/src/quick-latex.tsx:10]:
```suggestion
const latexUrlDark = "https://latex.codecogs.com/png.image?" + encodeURIComponent("\\dpi{512}\\bg{black}\\fg{white}");
```
Maybe change it a bit like this

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Hi 👋 

Thanks for your contribution 💪 

I have now tested your extension, and I have some feedback ready for you:

- `state.items` seems to be always empty, or? 

I'm looking forward to testing this extension again 🔥 

Request a new review when you are ready. Feel free to contact me here or at [Slack](https://www.raycast.com/community) if you have any questions.

**Comment by Noamko** [comment]
It appears that ﻿state.items is always empty, if my memory serves me correctly. I used a ﻿List initially as it seemed to be the most suitable container for the image. I'm not certain if there is a better solution now. any suggestions?

**Comment by pernielsentikaer** [comment]
I can't find a better solution that using a [List.EmptyView](https://developers.raycast.com/api-reference/user-interface/list#list.emptyview) for this, it seems like the right now that only an image will be shown

**Review by pernielsentikaer — APPROVED** [review]

**Pull request merged**